README.md

```diff
@@ -63,6 +63,10 @@ If you're looking to contribute to certain parts of the Chef server, familiarity
 - sqitch (database migrations)
 - lua (routing rules in openresty)
 
+## ACC Pipeline
+
+Chef Software, Inc. leverages an internal Chef Automate installation, commonly referred to as ACC, to "drink our own champagne" by allowing us to easily test and leverage builds of our products as they are created. The `chef/chef-server` repository, like many other Chef Software repositories, leverages an internal utility called Chef Expeditor to create a pub-sub model of actions across our various CI/CD utilities. One of those actions is creating creating a change in our internal Chef Automate instance every time we successfully generate a build of Chef Server. This change allows us to easily perform a complex matrix of test suites, followed by deployment to the internal Chef Server that we use to drive our internal infrastructure. While there is code to support this pipeline in this repository, this repository does not depend on Chef Automate to handle pull requests.
```

Comment:
[nit] `creating creating` (also, if this had some line breaks, I could point out the sentence 😉 )